beep/featurize.py

```diff
@@ -623,24 +624,24 @@ def features_from_processed_cycler_run(cls, processed_cycler_run):
         Vd = interpolated_df.voltage[interpolated_df.cycle_index == iini]
         Qd_diff = Qd_final.values - Qd_10.values
 
-        X[5] = np.log10(np.abs(np.min(Qd_diff)))  # Minimum
-        labels.append("abs_min_discharge_capacity_difference_cycles_2:100")
+        if len(Qd_diff):
```

Comment:
Could we include a comment here just as a reminder about why this logic exists?

Reply:
Done!